client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -42,6 +42,19 @@
             ng-show="_editable"
             translate>Publish</button>
 
+        <div class="navbtn dropdown pull-left" dropdown>
+            <button class="dropdown-toggle" dropdown-toggle><i class="icon-dots-vertical"></i></button>
+            <div class="dropdown-menu">
+                <ul>
+                    <li ng-controller="TemplateActionsController as template">
```

Comment:
Well, there's a lot of code elsewhere in the application that doesn't use `controllerAs` syntax and places everything into `$scope` instead. It's just about being more explicit in the template about where things come from, but as I said, it's not that big of a deal. :)